src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -96,6 +101,14 @@ private void setSubmissionData(Map<String, Object> data, String overseasEntityId
         data.put(MANAGING_OFFICERS_INDIVIDUAL_FIELD, submissionDto.getManagingOfficersIndividual());
         data.put(MANAGING_OFFICERS_CORPORATE_FIELD, submissionDto.getManagingOfficersCorporate());
         data.put(BENEFICIAL_OWNERS_STATEMENT, submissionDto.getBeneficialOwnersStatement());
+
+        // Convert trust data to JSON string
+        ObjectMapper mapper = JsonMapper.builder().findAndAddModules().build();
+        try {
+            data.put(TRUST_DATA, mapper.writeValueAsString(submissionDto.getTrusts()));
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
```

Comment:
I wouldn't use printStackTrace here, could you throw a ServiceException, you should be able to use the e.getMessage as message and pass the original exception e in as another param to ServiceException